packages/block-library/src/image/image.js

```diff
@@ -726,6 +778,10 @@ export default function Image( {
 		);
 	}
 
+	if ( ! url && ! temporaryURL ) {
+		return sizeControls;
```

Comment:
Do we need a new `sizeControls` const or can we reuse `controls`?

Reply:
`controls` includes caption and I think setting caption on placeholder is not expected.

Reply:
Could we update controls to extend sizeControls?

Reply:
Sure we could add some internal getter for controls that supports an include caption arg but why tho 😁 - it's not something really reused, it's a one time need in terms of the block's codebase.